src/modules/roc_pipeline/receiver.cpp

```diff
@@ -146,11 +146,18 @@ bool Receiver::create_session_(const packet::PacketPtr& packet) {
         roc_log(LogError, "receiver: can't create session, unexpected non-udp packet");
         return false;
     }
+
+    if (packet->udp()->dst_addr == config_.repair_port.address) {
+        roc_log(LogInfo, "receiver: can't create session, unexpected repair packet");
```

Comment:
We usually don't use LogInfo for per-packet messages because they could be quite frequent. I'd change it to LogDebug here.

Also, the log message is a bit unclear. The packet is not unexpected. Actually, it's quite expected, we just don't want to handle it. I'd change it to something like:

```
receiver: dropping repair packet for non-existent session
```

Reply:
455a825301accba5d3f79c7d45ef4315171d0a3f